.travis.yml

```diff
@@ -1,33 +1,15 @@
 sudo: required
 
 services:
-      - docker
-addons:
-  apt:
-    packages:
-      - docker-ce
-
-before_install:
+  - docker
 
 script:
-    - sh build.sh
-    # Invoke ci script too
-    - sh contrib/ci.sh
-after_success:
-    - if [ ! -s "$TRAVIS_TAG" ] ; then 
-        docker tag $DOCKER_NS/gateway:latest-dev $DOCKER_NS/gateway:$TRAVIS_TAG;
-        docker login -u=$DOCKER_USERNAME -p=$DOCKER_PASSWORD;
-        docker push $DOCKER_NS/gateway:$TRAVIS_TAG;
-        fi
+  - docker run --rm -it -v $PWD:/docs squidfunk/mkdocs-material build --verbose --clean --strict
+
 deploy:
-  provider: releases
```

Comment:
These elements cannot be deleted?

Reply:
I believe that these deletions were a bungled merge/merge conflict resolution. I'll put these changes back